virttest/utils_numeric.py

```diff
@@ -1,7 +1,7 @@
 import math
 
 
-def align_factor(value, factor=1024):
+def align_value(value, factor=1024):
```

Comment:
https://github.com/avocado-framework/avocado-vt/blob/7d3356e4f29af386c60123b83749eb1d77d8781b/virttest/utils_numeric.py#L10

we can also mention the return as aligned value.

Reply:
@balamuruhans Done. Changed the doc string of 'return'. Please have a look.